src/servers/ZoneServer2016/managers/worlddatamanager.ts

```diff
@@ -1155,7 +1156,8 @@ export class WorldDataManager {
         new Float32Array(entityData.rotation),
         server,
         getCurrentServerTimeWrapper().getTruncatedU32(),
-        entityData.vehicleId
+        entityData.vehicleId,
+        entityData?.shaderGroupId ?? 0
```

Comment:
Why "?" on entitydata?

Reply:
In case it doesn't exist, though probably unnecessary.